### PR TITLE
Restructure /in* pages

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: pages/in/bonus/index.vue [hunk 1]
```diff
@@ -165,12 +176,21 @@ export default {
...
       const user = this.getUserInfo;
       this.user = user.user;
       this.updateReferralStat();
+      this.updateLikeBonus();
+    },
+    async updateLikeBonus() {
+      try {
+        this.likeBonusAmountStr = await this.fetchUserTotalBonus(this.getUserInfo.user);
+      } catch (err) {
+        console.log(err);
```
Comment (williamchong007) [inline pages/in/bonus/index.vue]: `console.error(err);` for errors

File: layouts/in.vue [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            :md-label="$t('In.tab.overview')"
+            to="/in" />
+          <md-tab
+            id="bonus-tab"
+            :md-label="$t('In.tab.bonus')"
+            to="/in/bonus" />
+          <md-tab
+            id="history-tab"
+            :md-label="$t('In.tab.history')"
+            to="/in/bonus/history" />
```
Comment (williamchong007) [inline layouts/in.vue]: does `to` support vue-router syntax like ` { name: 'in-bonus' }`? prefer `name` if so

File: pages/in/bonus/index.vue (hunk 1 above)
Comment (nwingt) [reply pages/in/bonus/index.vue]: @williamchong007 Updated

File: layouts/in.vue (hunk 2 above)
Comment (nwingt) [reply layouts/in.vue]: Not support

**Comment by nwingt** [comment]
I added `in/edit` for redirection, is that enough?

File: components/header/HeaderWithMenuButton.vue [hunk 3]
```diff
@@ -66,7 +66,7 @@ export default {
       if (!this.getUserIsRegistered && this.showLogin) {
         this.showLoginWindow();
       } else {
-        this.$router.push({ name: this.getUserIsRegistered ? 'in-edit' : 'in-register' });
+        this.$router.push({ name: 'in' });
```
Comment (williamchong007) [inline components/header/HeaderWithMenuButton.vue]: should keep `in-register`?

File: components/header/Header.vue [hunk 4]
```diff
@@ -63,7 +63,7 @@ export default {
       if (!this.getUserIsRegistered && this.showLogin) {
         this.showLoginWindow();
       } else {
-        this.$router.push({ name: this.getUserIsRegistered ? 'in-edit' : 'in-register' });
+        this.$router.push({ name: 'in' });
```
Comment (williamchong007) [inline components/header/Header.vue]: should keep `in-register`?

**Comment by nwingt** [comment]
@williamchong007 Fixed unit testing

**Pull request merged**